### ISSUE TITLE
[お知らせ] マージ後の「作業ブランチ自動削除」を有効にしました

### ISSUE THREAD

**Issue by KEINOS** [Qithub-BOT/mastogetter#113]
リポジトリの以下の設定を有効にしました。

<kbd><img width="100%" alt="スクリーンショット 2020-01-16 11 32 56" src="https://user-images.githubusercontent.com/11840938/72488349-0e818900-3854-11ea-819c-d9907649affb.png"></kbd>

既読＆了解は 👍 お願いします。 👍 ×3 以上で Close します。

無効にした方がいい場合はコメントください。

**Comment by yumetodo** [comment]
あれっ、 #114 はそうならなかったけどどういうりくつなんだろうか

**Comment by KEINOS** [comment]
おりょ？本当だ。どういう理屈なんだろう？

原因がわかるまで Open にしておきます。

**Comment by sasanquaneuf** [comment]
Qithub-BOTのブランチは自動削除されるけど、ほかのひとのリポジトリのブランチは削除されないということでは
※ちなみに、わたしは昔Qithub-BOTに新しいブランチをpushしたらrejectされました

**Comment by yumetodo** [comment]
@sasanquaneuf それはMember権限がないからだと。

**Comment by sasanquaneuf** [comment]
@yumetodo はい、そうだろうなとは思っています。（だから自動削除される運用にしても現状のフローだとあんまり効果ないかもしれないですね、というのが言いたいことでした）
